src/app.ts

```diff
@@ -491,73 +515,49 @@ export class App {
           if (method.description) {
             out.comment(formatComment(method.description));
           }
-          const requestParameters: Record<
-            string,
-            gapi.client.discovery.JsonSchema
-          > = sortObject({ ...parameters, ...method.parameters });
-          const hasRequestRef = method.request && method.request['$ref'];
-          if (
-            !(requestParameters.hasOwnProperty('resource') && hasRequestRef)
-          ) {
-            // no resource param and no body at the same time -> generate x(request)
+
+          const requestRef = method.request?.$ref;
+          const requestParameters: Record<string, JsonSchema> = sortObject({
+            ...parameters,
+            ...method.parameters,
+          });
```

Comment:
The current state assumes `requestParameters` is not empty, which might not be the case?

Reply:
This _does_ seem to be not empty across the board, so I think this is fine for now.